src/openms/source/METADATA/SpectrumLookup.cpp

```diff
@@ -350,7 +350,14 @@ namespace OpenMS
         try
         {
           String value = String(matches[0]);
-          return value.toInt();
+          if (native_id_type_accession == "MS:1000774")
+          {
+            return ++(value.toInt()); // if the native ID is index=.., the scan number is usually considered index+1 (especially for pepXML)
```

Comment:
ok then probably this needs update: https://github.com/OpenMS/OpenMS/blob/c2555c3278de34a5f261df9a0915f42a8bfada3c/src/tests/class_tests/openms/source/SpectrumLookup_test.cpp#L177

Reply:
I think like you said. index is usually scan - 1.
"Index is the spectrum number in the file, starting from 0." From the mzTab-M specification.